### PR TITLE
Update README with code sample fixes

### PR DESCRIPTION
**Pull request by chad-ramos**
Code for sample repositoriesData (line 54) missing closing brackets.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,8 +52,8 @@ import eventDrops from 'event-drops';
 const chart = eventDrops({ d3 });
 
 const repositoriesData = [
-    { name: "admin-on-rest", data: [{ date: new Date('2014/09/15 14:21:31') }, /* ... */ ,]
-    { name: "event-drops", data: [{ date: new Date('2014/09/15 13:24:57') }, /* ... */ ,]
+    { name: "admin-on-rest", data: [{ date: new Date('2014/09/15 14:21:31') }, /* ... */ ,]},
```
Comment (jpetitcolas) [inline README.md]: Can you add a space before ending curly bracket?

**Comment by chad-ramos** [comment]
What are your thoughts on changing the following

` { name: "admin-on-rest", data: [{ date: new Date('2014/09/15 14:21:31') }, /* ... */ ,] },`

to

` { name: "admin-on-rest", data: [{ date: new Date('2014/09/15 14:21:31') }, /* ... */] },`

I am failing to see the value of ` ,` after `/* ... */` on lines 55 and 56

**Comment by jpetitcolas** [comment]
No problem for the last change. I guess it was on several lines before, but Prettier reviewed it. :)

**Comment by jpetitcolas** [comment]
Thanks! :)

**Pull request merged**